### ISSUE TITLE
node-notifier works for Windows 7 but this doesn't?

### ISSUE THREAD

**Issue by Jakobud** [mikaelbr/gulp-notify#121]
Using `node-notifier` works for Windows 7. But for some reason this plugin is based on `node-notifier` but it doesn't work for Windows 7. Am I misunderstanding something?

**Comment by mikaelbr** [comment]
Hi! Windows 7 doesn't have toaster-like notifications. But node-notifier should fallback to bouble-notifications (tooltip to the lower right corner usually). Or if you want more fully fledged notifications, you can use Growl.

**Comment by Jakobud** [comment]
Yes, `node-notifier` falls back to the bubble notifications when I use `node-notifier` directly in my project. But what I'm saying is that when I attempt to use `gulp-notify` I don't get any bubble notifications.
